core/src/test/scala/org/apache/spark/status/api/v1/SimpleDateParamSuite.scala

```diff
@@ -18,12 +18,13 @@ package org.apache.spark.status.api.v1
 
 import org.scalatest.{Matchers, FunSuite}
 
-class SimpleDateParamTest extends FunSuite with Matchers {
+class SimpleDateParamSuite extends FunSuite with Matchers {
 
   test("date parsing") {
     new SimpleDateParam("2015-02-20T23:21:17.190GMT").timestamp should be (1424474477190L)
-    new SimpleDateParam("2015-02-20T17:21:17.190CST").timestamp should be (1424474477190L)
-    new SimpleDateParam("2015-02-20").timestamp should be (1424390400000L)  // GMT
+    new SimpleDateParam("2015-02-20T17:21:17.190EST").timestamp should be (1424470877190L)
+    new SimpleDateParam("2015-02-20GMT").timestamp should be (1424390400000L)
```

Comment:
do you mind leaving the test case that when there is no timezone specified, GMT is assumed?  I figure that if there is going to be some default, thats the only thing that makes sense.  Or does the test not pass that way for you?

if you think there should *not* be any default, than we should change the behavior in `SimpleDateParam` (though I'd really prefer to leave it).